src/robusta/integrations/mail/sender.py

```diff
@@ -0,0 +1,198 @@
+import logging
+import tempfile
+from typing import List
+
+import apprise
+from apprise import NotifyFormat, NotifyType
+
+from robusta.core.reporting.base import BaseBlock, Emojis, Finding, FindingStatus
+from robusta.core.reporting.blocks import (
+    FileBlock,
+    LinksBlock,
+    LinkProp,
+    MarkdownBlock,
+)
+from robusta.core.reporting.consts import EnrichmentAnnotation
+from robusta.core.sinks.transformer import Transformer
+
+
+def with_attr(obj, attr_name, attr_value):
+    setattr(obj, attr_name, attr_value)
+    return obj
+
+
+class MailTransformer(Transformer):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.file_blocks = []
+
+    def block_to_html(self, block: BaseBlock) -> str:
+        # TODO should we additionally support ScanReportBlock here?
```

Comment:
Yes, I think we should
Any reason not to?
We can convert it to PDF and send it as a file attachment (like we do on Slack)
I think users will use it

Reply:
In fact it looks like the Slack sink is the *only* sink that handles ScanReportBlocks, so I was not sure if it's important at all. Will add.

Reply:
Added support for ScanReportBlocks via PDF files, just as in the Slack sink. Additionally added a check & a warning in `robusta.core.sinks.transformer`